src/main.rs

```diff
@@ -317,8 +317,26 @@ async fn main() {
     let num_workers = matches.value_of_t("num-workers").unwrap();
     let schema_registry = matches.value_of("schema-registry").unwrap();
 
-    let sr_settings = SrSettings::new(String::from(schema_registry));
-
+    info!("Using following settings:");
+    info!("  brokers:         {}", brokers);
+    info!("  group_id:        {}", group_id);
+    info!("  input_topic:     {}", input_topic);
+    info!("  output_topic:    {}", output_topic);
+    info!("  num_workers:     {}", num_workers);
+    info!("  schema_registry: {}", schema_registry);
+
+    let schema_registry_urls = schema_registry.split(",").collect::<Vec<&str>>();
+    let mut sr_settings_builder = SrSettings::new_builder(schema_registry_urls.first().unwrap().to_string());
+    for (i, url) in schema_registry_urls.iter().enumerate() {
+        if i > 0 {
+            sr_settings_builder.add_url(url.to_string());
+        }
```

Comment:
Kan bruke `skip` til å skippe det første elementet.
```suggestion
    for url in schema_registry_urls.iter().skip(1) {
        sr_settings_builder.add_url(url.to_string());
```